.github/ISSUE_TEMPLATE/release.md

```diff
@@ -10,44 +10,28 @@ Please check items along as you follow the release process.
 
 #### Overview
 
-This is a checklist for releases. This is filled in by both the releaser and the reviewer where necessary.
+This is a checklist for releases.
 
-#### Preparation
+#### Release candidate
 
 - [ ] Create a [documentation release issue](https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/new?title=Release+v3.x.x&labels=release&template=release.md).
-- [ ] Run `tools/bin/prepare_release minor` or `tools/bin/prepare_release patch`, depending on the kind of release.
-- [ ] Follow the instructions.
-
-#### Check 1 (for reviewers)
-
-- [ ] The Changelog is complete i.e., contains only the changes that are in the release (not more/less).
-- [ ] The version files are correctly updated.
+- [ ] Run the `create-release-branch.yml` Github Action.
+- [ ] Check that the versioning and `CHANGELOG.md` on the new branch are correct.
+- [ ] Run the `release-rc.yml` Github Action.
+- [ ] You can repeatedly run the `release-rc.yml` Github Action for multiple release candidates.
 
 #### Release
 
-- [ ] Once this PR is approved and merged, checkout the latest `v3.${minor}` branch locally.
-- [ ] Create a version tag
+- [ ] Checkout the latest `release/v3.${minor}.${patch}` branch locally.
```

Comment:
> Is it not possible to tag via an action too ?

This can be done, but we need some human to push the button "release this now" anyway, and whether it's via github.com, github CLI, git tag, or whatever, doesn't make any difference.

> Then the tagging action can also open a PR that merges the version increment in the original branch.

Ah yeah, I'll add this.